cszczepaniak/cribbly-backend
# Add game controller

##What I added
A Game controller that will create and grab individual game records

##Notes
-To correctly get all of the appropriate data, a new join table needed to be made called "Assignments". We can make a data model for this in the app if you want, but since it is used for nothing but looking up and mapping, it might not be necessary

##How to test it
-Make 4 players
-Make 2 teams
-Run the `Create` endpoint, which should return an OK status
-Run the `GetById` endpoint to see that the players and teams have been correctly populated into the response
